packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -15,9 +16,25 @@ export const SUCCESS_ICON = 'clipboard-checked';
 export const ERROR_ICON = 'clipboard-x';
 export const DEFAULT_STATUS = 'idle';
 
-export default class HdsCopyButtonComponent extends Component {
+export interface HdsCopyButtonSignature {
+  Args: {
```

Comment:
Probably you could `Pick` a few properties from `ButtonSignature` (`text`, `isFullWidth`, `isIconOnly`) and remove them from here.

Reply:
Same for the properties from `hds-clipboard`

Reply:
I'm having trouble figuring out how to import the properties from the `hds-clipboard`

Reply:
Let's sync later